_custombuild

```diff
@@ -8,7 +8,10 @@
  */
 var gulp = require('gulp');
 var merge = require('gulp-merge');
+var uglify = require('gulp-uglify');
 var replace = require('gulp-replace');
+var wrapper = require('gulp-wrapper');
+var stripComments = require('gulp-strip-comments');
```

Comment:
Oops. I forgot to remove those. I was trying to find the best way to get rid of comments which ended up just being a regex.